packages/harmony/package.json

```diff
@@ -15,6 +15,8 @@
     "test": "test-storybook",
     "lint": "eslint src --ext=ts,tsx",
     "lint:fix": "npm run lint -- --fix",
+    "stylelint": "stylelint 'src/**/*.css'",
+    "stylelint:fix": "npm run stylelint -- --fix",
```

Comment:
Awesome! While we're here, let's add these too so it gets picked up by `npm run verify` from root
```
"typecheck": "tsc",    
"verify": "concurrently \"npm:typecheck\" \"npm:lint:fix\"",
```

Reply:
yes!